unit-tests/log/test-c-debug.cpp

```diff
@@ -24,8 +24,12 @@ TEST_CASE( "Logging C DEBUG", "[log]" ) {
 
     rs2_error* e = nullptr;
     rs2_log_to_callback( RS2_LOG_SEVERITY_DEBUG, c_callback, nullptr, &e );
+#if BUILD_EASYLOGGINGPP
```

Comment:
Can we arrange it so all these tests aren't even compiled and run if ELPP is off? We can make a single new test that tests that log_to_callback throws, instead